recipes/cufflinks/build.sh

```diff
@@ -5,6 +5,36 @@ mkdir -p $PREFIX/bin
 if [ $PY3K -eq 1 ]
 then
     2to3 --write cuffmerge
+    patch -p0 <<EOF
+--- cuffmerge	2015-11-13 11:08:06.630776225 +0100
++++ cuffmerge-fixed	2015-11-13 11:15:43.444587251 +0100
+@@ -158,7 +158,7 @@
+         pass
+     else:
+         os.mkdir(tmp_root)
+-    return tmp_root + prefix + os.tmpnam().split('/')[-1]
++    return tempfile.NamedTemporaryFile(dir=tmp_root, prefix=prefix).name
+
+ def cufflinks(out_dir,
+               sam_file,
+@@ -485,7 +485,7 @@
+ def header_for_chrom_info(chrom_info):
+     header_strs = ["""@HD\tVN:1.0\tSO:coordinate"""]
+     chrom_list = [(chrom, limits) for chrom, limits in chrom_info.items()]
+-    chrom_list.sort(lambda x,y: cmp(x[0],y[0]))
++    chrom_list.sort(key=lambda x: x[0])
+     #print chrom_list
+     for chrom, limits in chrom_list:
+         line = "@SQ\tSN:%s\tLN:\t%d" % (chrom, limits[1])
+@@ -523,7 +523,7 @@
+         start_time = datetime.now()
+         prepare_output_dir()
+
+-        run_log = open(logging_dir + "run.log", "w", 0)
++        run_log = open(logging_dir + "run.log", "w")
+         run_cmd = " ".join(argv)
+         print(run_cmd, file=run_log)
+EOF
```

Comment:
Thanks for you work on bioconda, instead!
The patch is needed only if the interpreter is python3 (maybe it works on python2 too, but maybe it is better to not modify the original code here), while the meta.yaml file would apply it in any case, wouldn't it?
I don't think I will be able to make big contributions, but I would be glad.